src/docs/ftw-analytics/template-sitemap/index.md

```diff
@@ -0,0 +1,110 @@
+to specify them here.
+
+### sitemap-index.xml
+
+The [your-domain]**/sitemap-index.xml** file further exposes three
+separate sub-sitemaps. This is useful because it allows you more
+detailed tracking in analytics tools, and it reduces the size of each
+individual sitemap.
+
+The index and the sub-sitemaps are generated in
+[server/resources/sitemap.js](https://github.com/sharetribe/web-template/blob/main/server/resources/sitemap.js).
+
+### sitemap-default.xml
+
+The [your-domain]**/sitemap-default.xml** file includes your
+marketplace's public built-in pages.
+
+It also includes the landing page, the terms of service page, and the
+privacy policy page – these pages have specified paths in the default
+template, even though their content is fetched from assets.
+
+If you add public custom paths to your template, remember to
+[add them in this sitemap](https://github.com/sharetribe/web-template/blob/main/server/resources/sitemap.js#L45)
+as well. You can also add sitemap entries for some of your key searches,
+e.g.
+
+```diff
+  const defaultPublicPaths = {
+    landingPage: { url: '/' },
+    termsOfService: { url: '/terms-of-service' },
+    privacyPolicy: { url: '/privacy-policy' },
+    signup: { url: '/signup' },
+    login: { url: '/login' },
+    search: { url: '/s' },
++   searchMountainBikes: { url: '/s?pub_category=mountainBike' }
+  };
+```
+
+### sitemap-recent-listings.xml
+
+The [your-domain]**/sitemap-recent-listings.xml** file shows the URLs
+for your marketplace's most recently created listings. The listing data
+is fetched from the _sdk.sitemapData.queryListings()_ endpoint, and the
+endpoint returns the 10 000 most recent public listings.
+
+The recent listing ids are mapped to the template's default canonical
+listing URLs. If you make changes to the listing canonical URL
+structure, be sure to make the corresponding changes to this sitemap as
+well.
+
+### sitemap-recent-pages.xml
+
+The [your-domain]**/sitemap-recent-pages.xml** file shows the URLs for
+your marketplace's asset-based pages that are shown in
+_[your-domain]/p/:pathId_. The page information is fetched using the
+_sdk.sitemapData.queryAssets()_ endpoint.
+
+The default logic filters out the pages that have custom paths: landing
+page, terms of service, and privacy policy. If you add public permanent
+paths for asset-based pages to your template and your
+_sitemap-default.xml_ file, be sure to filter them out of the SDK result
+set in
+[server/resources/sitemap.js](https://github.com/sharetribe/web-template/blob/main/server/resources/sitemap.js#L284)
+
+## Sitemap caching
+
+The template caches sitemap data
+[for one day by default](https://github.com/sharetribe/web-template/blob/main/server/resources/sitemap.js#L55).
+Especially on bigger sites with multiple listings, listing sitemap data
+can take up to a few seconds to fetch from the SDK. Caching the results
+improves the sitemap performance.
+
```

Comment:
Also, API will cache listings & pages endpoint. Currently, it's 1 day too, which means that the combined caching effect might be a maximum of 2 days on top of the sharetribe/web-template.